components/loginForm/style.ts

```diff
@@ -0,0 +1,21 @@
+import {StyleSheet} from 'react-native';
+export const styles = StyleSheet.create({
+  sectionContainer: {
+    marginTop: 32,
+    paddingHorizontal: 24,
+    fontSize: 26,
+    textAlign: 'center',
+  },
+  logInBtn: {
+    marginTop: 52,
+    marginLeft: 12,
+    marginRight: 12,
```

Comment:
for left & right, alternatively use `marginHorizontal`
likewise for top & bottom, we can use `marginVertical`